src/main/webapp/go.html

```diff
@@ -16,9 +16,9 @@
       <div class="center-text" id="map-message-area"></div>
       <form action="/guess-data" method="POST">
         <input type="text" name="guess-input" size="50"/>
-        <input type="submit" class="button" id="submit-button"/>
+        <input type="submit" class="submit-button" id="submit-button"/>
```

Comment:
FYI I think you could replace the `input` tag with a `button`, so that it could automatically be styled by the CSS without having an extra class, like so:
```suggestion
        <button type="submit"/>
```
I'm fine with leaving this as-is though. Just sharing the info.